compiler-cli/src/new.rs

```diff
@@ -11,11 +11,11 @@ use strum::{Display, EnumString, EnumVariantNames};
 
 use crate::NewOptions;
 
-const GLEAM_STDLIB_VERSION: &str = "0.25";
-const GLEEUNIT_VERSION: &str = "0.7";
-const ERLANG_OTP_VERSION: &str = "25.1";
-const REBAR3_VERSION: &str = "25.1";
-const ELIXIR_VERSION: &str = "1.14.1";
+const GLEAM_STDLIB_VERSION: &str = "0.25.0";
+const GLEEUNIT_VERSION: &str = "0.7.2";
+const ERLANG_OTP_VERSION: &str = "25.2";
+const REBAR3_VERSION: &str = "3.20.0";
```

Comment:
ok

Reply:
...it is not set to `"3";` in case you meant `3.0` or `3.0.0`.

Proof:
https://github.com/inoas/fix_and_update_gleam_new_deps_test/actions/runs/3707684520/jobs/6284391589
https://github.com/inoas/fix_and_update_gleam_new_deps_test/blob/main/.github/workflows/test.yml